ci/ubuntu-setup-lld.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+LLVM_VERSION=13
+
+codename="$(grep '^VERSION_CODENAME=' /etc/os-release | sed 's/^VERSION_CODENAME=//')"
+echo "deb http://apt.llvm.org/${codename}/ llvm-toolchain-${codename}-13 main" |
+    sudo dd of="/etc/apt/sources.list.d/llvm-toolchain-${codename}-13.list"
```

Comment:
This is the first time to see `dd` is used in such a case. Do you have some examples?

Reply:
I recommend using `tee` instead of `dd` if `tee` is already installed.